assignment/api/src/resources/assignment_eu.properties

```diff
@@ -1091,3 +1090,18 @@ search.label=Bilaketa
 search.button=Bilaketa
 search.clear=Garbitu bilaketa
 
+peerAssessmentStudents=Los estudiantes se eval\u00faan entre ellos
+peerAssessmentGroups=Los grupos se eval\u00faan entre ellos
+peerassessment.group=Grupo {0}
+option.selfreport=Permitir que los estudiantes autoevaluen su tarea
+option.selfreport.must=Los estudiantes deben completar entera su autoevaluaci\u00F3n para poder realizar el env\u00EDo
+option.selfreport.partialmust=Los estudiantes deben completar al menos un criterio de su autoevaluaci\u00F3n para poder realizar el env\u00EDo
+option.selfreport.notamust=Los estudiantes no tienen porqu\u00E9 completar la autoevaluaci\u00F3n para poder realizar el env\u00EDo
+youmustrubric=Esta tarea es autoevaluable. Puedes autoevaluar tu tarea con la siguiente r\u00fabrica antes de enviarla.
+youhavetorubric=Esta tarea es autoevaluable. Debes autoevaluar tu tarea con la siguiente r\u00fabrica antes de enviarla. Debes completar entera la r\u00fabrica para realizar el env\u00EDo.
+youhavetorubricone=Esta tarea es autoevaluable. Debes autoevaluar tu tarea con la siguiente r\u00fabrica antes de enviarla. Debes completar al menos un criterio de la r\u00fabrica para realizar el env\u00EDo.
+studentrubric=Esta tarea es autoevaluable. Puedes revisar la autoevaluaci\u00f3n del estudiante antes de puntuar la tarea.
+autoevaluation=Autoevaluaci\u00f3n:
+instructor_grading=Correcci\u00f3n del profesor:
+reviewrubric=Esta tarea es autoevaluable. Puedes revisar tu autoevaluaci\u00f3n antes de enviarla.
+reviewrubricreport=Esta es tu autoevaluaci\u00f3n.
```

Comment:
@PabloSanRoman could you please provide the translations for this? Thanks!

Reply:
Hi @bgarciaentornos , here is the translation: 

peerAssessmentStudents= Ikasleek elkarri ebaluatuko diete. 
peerAssessmentGroups= Taldeek elkarri ebaluatuko diete. 
peerassessment.group=Taldea {0}
option.selfreport=Baimendu autoebaluazioa
option.selfreport.must=Ikasleek autoebaluazio osoa osatu behar dute bidalketa egin ahal izateko.
option.selfreport.partialmust=Ikasleek beren autoebaluazioaren irizpide bat bete behar dute gutxienez, bidalketa egin ahal izateko.
option.selfreport.notamust=Ikasleek ez dute zertan autoebaluazioa osatu behar bidalketa egin ahal izateko.
youmustrubric=Zeregin hau autoebaluagarria da. Zure zeregina autoebaluatu dezakezu honako errubrika honekin bidali baino lehen.
youhavetorubric=Zeregin hau autoebaluagarria da. Zure zeregina autoebaluatu behar duzu honako errubrika honekin bidali baino lehen. Errubrika osoa bete behar duzu bidalketa egiteko.
youhavetorubricone=Zeregin hau autoebaluagarria da. Zure zeregina autoebaluatu behar duzu honako errubrika honekin bidali baino lehen. Gutxienez errubrikaren irizpide bat bete behar duzu bidalketa egiteko.
studentrubric=Zeregin hau autoebaluagarria da. Ikaslearen autoebaluazioa berrikusi dezakezu zeregina puntuatu baino lehen.
autoevaluation=Autoebaluazioa: 
instructor_grading=Irakaslearen zuzenketa:
reviewrubric=Zeregin hau autoebaluagarria da. Zure ebaluazioa berrikusi dezakezu bidali baino lehen.
reviewrubricreport=Hau da zure ebaluazioa.